test/proto_client_test.dart

```diff
@@ -31,11 +29,11 @@ main() async {
 
   peer.on('request', (request, accept, reject) {
     print('request: ' + request.toString());
-    accept({ 'key1':"value1", 'key2':"value2"});
+    accept({'key1': "value1", 'key2': "value2"});
     //reject(404, 'Oh no~~~~~');
   });
 
-  await peer.connect();
+  //await peer.connect();
```

Comment:
Why is this commented out? Does the test run?

Reply:
Oh my bad I was playing around with the code and might have commented it out and forgot to change it back by mistake.

Reply:
![image](https://user-images.githubusercontent.com/51456744/129902092-07533481-df95-46bf-ab09-92bdf3e94082.png)

Looks like i got mixed up in what I commented, this was done by you so I am not sure about the reason 😅

Reply:
ah lol. Okay, yeah the PR looks good. It's usable, code quality shuldn't really be a priority here anyway. If someone is willing enough to cleanup and optimize so be it. But this should be good enough for pub.dev